project.clj

```diff
@@ -48,7 +48,9 @@
       [lein-ancient                    "0.6.10"]
       ;; [com.cemerick/austin          "0.1.4"]
       [com.cemerick/clojurescript.test "0.3.3"]
-      [lein-codox                      "0.9.5"]]}]}
+      [lein-codox                      "0.9.5"]
+      [lein-npm                        "0.6.2"]]}]}
+  :npm {:dependencies [[websocket "1.0.23"]]}
```

Comment:
This NPM dep does leak downstream: https://github.com/RyanMcG/lein-npm/pull/13. To test this, add the `lein-npm` plugin to the example project and run `lein deps`. You'll see `lein-npm` installs `websocket`. `lein-npm` looks into a projects dependencies to see if any of them have any `:npm` dependencies to install too. I'd probably be inclined to remove this dependency entirely here and just make a note of it as being required for Node users to add it to their own projects?